web/src/graphql/mutations/login.graphql

```diff
@@ -0,0 +1,12 @@
+mutation Login($options: UsernamePasswordInput!) {
+  login(options: $options) {
```

Comment:
Note: we can use the options type instead of manually defining and passing the username/password properties of options, and they're already autogenerated by codegen ✨